servers/rendering/shader_language.cpp

```diff
@@ -8640,7 +8640,11 @@ Error ShaderLanguage::_parse_shader(const HashMap<StringName, FunctionInfo> &p_f
 							completion_line = tk.line;
 
 							if (!is_token_hint(tk.type)) {
-								_set_error(RTR("Expected valid type hint after ':'."));
+								if (tk.text == "hint_color") {
+									_set_error(RTR("hint_color is no longer supported, use source_color instead."));
```

Comment:
```suggestion
									_set_error(RTR("`hint_color` is no longer supported, use `source_color` instead."));
```

Reply:
@Kalrog Bump :slightly_smiling_face: 

After applying the suggestion, remember to squash the commits together so we can merge this. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions :slightly_smiling_face: